<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroUsuario</class>
 <widget class="QWidget" name="RegistroUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>51</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>341</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nombreLabel">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nombreLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="apellidoLabel">
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="apellidoLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="eMailLabel">
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="eMailLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="contrasenaLabel">
      <property name="text">
       <string>Contraseña</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="contrasenaLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="repitaContrasenaLabel">
      <property name="text">
       <string>Repita Contraseña</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="repitaContrasenaLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="tipoDeUsuarioLabel">
      <property name="text">
       <string>Tipo de Usuario</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="tipoDeUsuarioComboBox"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="codigoDeAdministradorLabel">
      <property name="text">
       <string>Codigo de Administrador</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="codigoDeAdministradorLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="aceptarPushButton">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelarPushButton">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
